src/plugins/intel_gpu/src/plugin/remote_tensor.cpp

```diff
@@ -28,20 +40,24 @@ RemoteTensorImpl::RemoteTensorImpl(RemoteContextImpl::Ptr context,
     , m_mem(mem)
     , m_surf(surf)
     , m_plane(plane) {
-    if (supports_caching()) {
-        m_hash = cldnn::hash_combine(0, m_mem);
-        m_hash = cldnn::hash_combine(m_hash, m_surf);
-        m_hash = cldnn::hash_combine(m_hash, plane);
-        m_hash = cldnn::hash_combine(m_hash, m_shape.size());
-        m_hash = cldnn::hash_combine(m_hash, element_type.hash());
-        for (const auto& d : m_shape) {
-            m_hash = cldnn::hash_combine(m_hash, d);
-        }
-    }
-
-    update_strides();
+    update_hash();
     allocate();
-    init_properties();
+}
+
+RemoteTensorImpl::RemoteTensorImpl(RemoteContextImpl::Ptr context,
+                                   const ov::Shape& shape,
+                                   const ov::element::Type& element_type,
+                                   cldnn::memory::ptr memory)
+    : m_context(context)
+    , m_element_type(element_type)
+    , m_shape(shape)
+    , m_memory_object(memory)
+    , m_layout(make_layout(element_type, shape))
+    , m_mem_type(allocation_type_to_tensor_type(memory->get_allocation_type()))
+    , m_mem(nullptr)
+    , m_surf(0)
+    , m_plane(0) {
+    update_hash();
```

Comment:
Aren't `update_stride()` and `update_properties()` needed here?

Reply:
Yeah, looks like they are needed. By the way, do you have any plans for this c-tr? Currently it's not used anywhere, maybe we can remove it at all?

Reply:
> By the way, do you have any plans for this c-tr?

I used it here https://github.com/vladimir-paramuzov/openvino/blob/dyn_memory_opt/src/plugins/intel_gpu/src/plugin/sync_infer_request.cpp#L651 

Though you can remove it from this PR and we'll add it later if it's really needed.

Reply:
@vladimir-paramuzov , I see, thank you

I have removed it for now